torchgeo/datasets/mapinwild.py

```diff
@@ -0,0 +1,328 @@
+import os
+import shutil
+
+from torch import Tensor
+from typing import Callable, Optional, Dict, Tuple
+import rasterio
+import numpy as np
+import torch
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from .geo import NonGeoDataset
+from .utils import check_integrity, extract_archive, download_url
+
+"""MapInWild dataset."""
+
+class MapInWild(NonGeoDataset):
+    """MapInWild dataset.
+
+        The `MapInWild <https://arxiv.org/abs/2212.02265>` is curated for the task of wilderness mapping on a pixel-level. 
```

Comment:
This is indented too far. Also, link is missing an underscore:
```suggestion
        The `MapInWild <https://arxiv.org/abs/2212.02265>`_ is curated for the task of wilderness mapping on a pixel-level. 
```
All lines should be less than 89 chars to make flake8 happy.